fastly/s3_test.go

```diff
@@ -27,10 +27,12 @@ func TestClient_S3s(t *testing.T) {
 			GzipLevel:                    9,
 			Format:                       "format",
 			FormatVersion:                2,
+			ResponseCondition:            "",
```

Comment:
Explicitly setting zero value just to show that it is covered.

However without adding additional setup --- i.e., creating a valid response condition --- we can't set a value here.

Maybe a comment?

Reply:
The coverage here seems fine to me. If you think a comment will help highlight your concern in the future, go for it.